File: Backend/src/main/java/com/futurum/task/models/Campaign.java
package com.futurum.task.models;

import lombok.*;

import javax.persistence.*;
import java.math.BigDecimal;
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@ToString
@Entity
@Table(name = "Campaigns")
public class Campaign {
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Id
    private Long campaignID;
    private String campaignName;
    private String keyWords;
    private int minAmount;
    private BigDecimal campaignFund;
    private boolean status = false;
    private String townName;
    private double radius;
}

File: Frontend/futurumTask/src/app/classes/campaign.ts
export class Campaign {
    campaignID : number;
    campaignName : string;
    keyWords : string;
    minAmount : number;
    campaignFund : number;
    status: boolean;
    townName : string;
    radius : number;

    constructor(campaignID : number,
        campaignName : string,
        keyWords : string,
        minAmount : number,
        campaignFund : number,
        status: boolean,
        townName : string,
        radius : number){
            this.campaignID = campaignID;
            this.campaignName = campaignName;
            this.keyWords = keyWords;
            this.minAmount = minAmount;
            this.campaignFund = campaignFund;
            this.status = status;
            this.townName = townName;
            this.radius = radius;
        }
}
